fulfilment_count.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+import csv
+from getpass import getpass
+
+import psycopg2
+
+from config import Config
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description='Tool to get fulfilment counts ')
+    parser.add_argument('fulfilment_date_from', help='From date e.g. 2019-10-18T16:00:00+01:00', type=str)
+    parser.add_argument('fulfilment_date_to', help='To date e.g. 2019-10-22T16:00:00+01:00', type=str)
+    parser.add_argument('username', help='Username to connect to database', type=str)
+    return parser.parse_args()
+
+
+def fulfilment_query(fulfilment_date_from, fulfilment_date_to, username, password):
+    sql_query = """SELECT event_payload ->> 'fulfilmentCode' AS fulfilment_code,
+     count(*)
+     FROM casev2.event
+      WHERE rm_event_processed BETWEEN %s AND %s
+      AND event_type = 'FULFILMENT_REQUESTED'
+      AND event_payload ->> 'fulfilmentCode' LIKE 'P_%%'
+      GROUP BY event_payload ->> 'fulfilmentCode';
+      """
+    conn = psycopg2.connect(f"dbname='{Config.DB_NAME}' user={username} host='{Config.DB_HOST}' "
+                            f"password={password} port='{Config.DB_PORT}'{Config.DB_USESSL}")
+    cur = conn.cursor()
+    cur.execute(sql_query, (fulfilment_date_from, fulfilment_date_to,))
+    db_result = cur.fetchall()
+
+    with open(f'fulfilment-{fulfilment_date_to[:10]}.csv', 'w') as out:
```

Comment:
Worth putting the `from` date in the filename as well to avoid file name clashes for different start dates with the same end?

Reply:
The naming convention at the moment is the same as what we are currently doing